### DISCUSSION TITLE
Missing Dependency from rsconnect when deploying shinyapps

### DISCUSSION THREAD

**Discussion by rafiff23** [teamalgoritma/community#727] [Q&A] [answered]
saya menemukan error ini ketika ingin deploy apps di shiny
```
Error: Unhandled Exception: Child Task 1319228024 error: Unhandled Exception: Missing dependency 'renv' for package 'rsconnect'
Execution halted
```
saya sudah mencoba untuk reinstall rsconnect dan mengganti project directory tetapi masalah yang sama masih muncul juga

**Comment by MF-Faqih** [comment, marked as answer]
apakah sudah dicoba untuk menginstall package `renv` ?
